forms-flow-web/src/services/UserService.js

```diff
@@ -81,10 +81,20 @@ const refreshToken = (store) => {
       console.log(error);
       userLogout();
     });
-  }, 6000);
+  }, checkExpireTime(KeycloakData));
 }
 
 
+const checkExpireTime =(keycloak)=>{
+  const {exp, iat} = keycloak.idTokenParsed
+  if(exp&&iat){
+    const toeknExpiretime =new Date(exp).getMilliseconds()-new Date(iat).getMilliseconds()
```

Comment:
* correct as tokenExpireTime try use camelCase
* Check if the expiration time is changed in between a session from keycloak will the new expiration time get in effect immediately.